### ISSUE TITLE
Mp3ToVec Skips my MP3s

### ISSUE THREAD

**Issue by dariusp1** [teticio/Deej-A.I.#12]
I can't seem to figure out why?

last 10 lines:

Skipping /Users/dariuspleasant/Documents/Music/Music 2/79rs Gang/Expect The Unexpected/07 79rs Gang and Lakou Mizik- Iko Kreyòl (79rs Version).mp3
100%|█████████████████████████████████████| 1362/1362 [00:20<00:00, 65.92file/s]
Creating MP3ToVecs for batch 1/1
Precalculating cosine distances
0vector [00:00, ?vector/s]
Calculating IDF weights
0vector [00:00, ?vector/s]
Calculating TF weights
0mp3 [00:00, ?mp3/s]

---

pip freeze

absl-py==0.13.0
astunparse==1.6.3
audioread==2.1.9
Brotli==1.0.9
cachetools==4.2.2
certifi==2021.5.30
charset-normalizer==2.0.3
click==8.0.1
dash==0.42.0
dash-core-components==0.47.0
dash-daq==0.1.4
dash-html-components==0.16.0
dash-renderer==0.23.0
dash-table==3.6.0
dataclasses==0.8
decorator==5.0.9
Flask==1.0.2
Flask-Compress==1.10.1
gast==0.3.3
google-auth==1.33.1
google-auth-oauthlib==0.4.4
google-pasta==0.2.0
grpcio==1.39.0
h5py==2.10.0
idna==3.2
importlib-metadata==4.6.1
itsdangerous==2.0.1
Jinja2==3.0.1
joblib==1.0.1
Keras==2.3.1
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.2
librosa==0.6.3
llvmlite==0.31.0
Markdown==3.3.4
MarkupSafe==2.0.1
mutagen==1.42.0
numba==0.48.0
numpy==1.16.3
oauthlib==3.1.1
opt-einsum==3.3.0
pandas==0.24.2
Pillow==8.3.1
plotly==5.1.0
protobuf==3.17.3
pyasn1==0.4.8
pyasn1-modules==0.2.8
python-dateutil==2.8.2
pytz==2021.1
PyYAML==5.4.1
requests==2.26.0
requests-oauthlib==1.3.0
resampy==0.2.2
rsa==4.7.2
scikit-learn==0.24.2
scipy==1.4.1
six==1.16.0
spotipy==2.4.4
tenacity==8.0.1
tensorboard==2.2.2
tensorboard-plugin-wit==1.8.0
tensorflow==2.2.0
tensorflow-estimator==2.2.0
termcolor==1.1.0
threadpoolctl==2.2.0
tqdm==4.31.1
typing-extensions==3.10.0.0
urllib3==1.26.6
Werkzeug==2.0.1
wrapt==1.12.1
zipp==3.5.0

**Comment by dariusp1** [comment]
What does this line in the code mean exactly? 

 if y.shape[0] < slice_size:
                            print(f'Skipping {full_path}')
                            continue
                        S = librosa.feature.melspectrogram(y=y, sr=sr, n_fft=n_fft, hop_length=hop_length, n_mels=n_mels, fmax=sr/2)
                        x = np.ndarray(shape=(S.shape[1] // slice_size, n_mels, slice_size, 1), dtype=float)
                        for slice in range(S.shape[1] // slice_size):
                            log_S = librosa.power_to_db(S[:, slice * slice_size : (slice+1) * slice_size], ref=np.max)
                            if np.max(log_S) - np.min(log_S) != 0:
                                log_S = (log_S - np.min(log_S)) / (np.max(log_S) - np.min(log_S))
                            x[slice, :, :, 0] = log_S
                        pickle.dump((full_path, model.predict(x)), open(dump_directory + '/' + pickle_filename, 'wb'))
                    except KeyboardInterrupt:
                        raise
                    except:
                        print(f'Skipping {full_path}')
